geomesa-quickstart-hbase/src/main/java/com/example/geomesa/hbase/HBaseQuickStart.java

```diff
@@ -0,0 +1,249 @@
+package com.example.geomesa.hbase;
+
+import com.beust.jcommander.internal.Lists;
+import com.google.common.base.Joiner;
+import com.vividsolutions.jts.geom.Geometry;
+import org.apache.commons.cli.*;
+import org.geotools.data.*;
+import org.geotools.factory.Hints;
+import org.geotools.feature.DefaultFeatureCollection;
+import org.geotools.feature.FeatureCollection;
+import org.geotools.feature.FeatureIterator;
+import org.geotools.feature.SchemaException;
+import org.geotools.feature.simple.SimpleFeatureBuilder;
+import org.geotools.filter.text.cql2.CQL;
+import org.geotools.filter.text.cql2.CQLException;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.locationtech.geomesa.utils.text.WKTUtils$;
+import org.opengis.feature.Feature;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.opengis.filter.Filter;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Copyright 2016 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+public class HBaseQuickStart {
+    static String TABLE_NAME = "bigtable.table.name".replace(".", "_");
```

Comment:
The parameter for ``HBaseDataStore`` is ``bigtable.table.name``, which in turn follows the conventions used by the HBase configuration parameters. This changes the argument switch to ``--bigtable_table_name``, because it gets confused by the periods. There are a couple of options:
* Map value of ``--table_name`` switch to ``bigtable.table.name`` parameter: Shorter command line, but makes the actual parameters for the data store less clear.
* Change name of parameter ``bigtable.table.name`` to ``table_name``: Requires a corresponding change in GeoMesa code; and since the parameter list is small, moving to a more generic parameter name increases the risk of ``DataStoreFinder.getDataStore()`` choosing the wrong store.